src/UFS2ARCO/ufsdataset.py

```diff
@@ -318,7 +321,11 @@ class FV3Dataset(UFSDataset):
         "grid_xt": 30,
     }
 
-    def open_dataset(self, cycle, fsspec_kwargs=None, **kwargs):
+    def __init__(self, *args, **kwargs):
+        super(FV3Dataset, self).__init__(*args, **kwargs)
+        self.zarr_name = "fv3.zarr"
```

Comment:
```suggestion
        self.zarr_name = "fv3.zarr"
        if len(self.chunks_in) == 0:
            self.chunks_in = {
                "pfull": 5,
                "grid_yt": -1,
                "grid_xt": -1,
            }
            chunks_out = {
                "time": 1,
                "pfull": 5,
                "grid_yt": 30,
                "grid_xt": 30,
            }
```

Reply:
My suggestion is to add the default for chunks_in and chunks_out here, and delete the attribute above as you did for zarr_name

Reply:
After that I don't have any additional comments.